tslint.json

```diff
@@ -63,6 +64,7 @@
     "no-trailing-whitespace": true,
     "no-unnecessary-initializer": true,
     "no-unused-expression": true,
+    "no-unused-variable": [true, {"ignore-pattern": "^_"}],
```

Comment:
~~short comment? (I guess the effect is that variables starting with an underscore will never be marked as unused by the linter, right? But why do we need this?)~~
ok, forget that, its JSON, so no comments, I guess…